src/notification/notification.stories.ts

```diff
@@ -1,12 +1,69 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
-import { action } from "@storybook/addon-actions";
+import { action, actions } from "@storybook/addon-actions";
 import { withKnobs, boolean, object } from "@storybook/addon-knobs/angular";
 
-import { Component } from "@angular/core";
+import { Component, Testability, Input } from "@angular/core";
```

Comment:
I don't think we need `Testability` here 🤔 

```suggestion
import { Component, Input } from "@angular/core";
```